packages/button/src/components/RainbowButton.tsx

```diff
@@ -0,0 +1,23 @@
+import { RainbowKitProvider, WalletButton } from '@rainbow-me/rainbowkit';
+import type { RainbowKitProviderProps } from '@rainbow-me/rainbowkit/dist/components/RainbowKitProvider/RainbowKitProvider';
+import React from 'react';
+
+export function RainbowButtonProvider({
+  children,
+  ...options
+}: Omit<RainbowKitProviderProps, 'chains'>) {
+  // NOTE: The chains array is intentionally left empty.
+  // This is due to the current incompatibility between the 'rainbowkit' chain modal and the 'rainbow' button library.
+  // As a workaround, users can utilize 'wagmi' to switch chains.
+  return (
+    <RainbowKitProvider chains={[]} {...options}>
```

Comment:
Basically we don't need `chains` array since we are not using the rainbowkit chain modal UI here. Ok to leave it empty for now. Since our focus with rainbow button is to only do `connect`.